frame/uniques/src/impl_nonfungibles.rs

```diff
@@ -142,6 +142,31 @@ impl<T: Config<I>, I: 'static> Mutate<<T as SystemConfig>::AccountId> for Pallet
 			Ok(())
 		})
 	}
+
+	fn set_collection_attribute(
+		collection: &Self::CollectionId,
+		key: &[u8],
+		value: &[u8],
+	) -> DispatchResult {
+		let bounded_key: BoundedVec<u8, T::KeyLimit> =
+			key.clone().to_vec().try_into().map_err(|_| Error::<T, I>::KeyTooLong)?;
+		let bounded_value: BoundedVec<u8, T::ValueLimit> =
```

Comment:
Could you please add some tests that these errors are triggered?  
Same for the `set_attribute`.

Reply:
https://github.com/paritytech/substrate/pull/11544/commits/ec38c47d55d4a4db1ea8e838b39cbe5bd5f5935b